tailwind.config.js

```diff
@@ -10,6 +10,15 @@ export default {
 			colors: {
 				primary: colors.red
 			}
+		},
+		keyframes: {
+			slideIn: {
+				'0%': { transform: 'translateX(-30%)' },
+				'100%': { transform: 'translateX(0)' }
+			}
+		},
+		animation: {
+			slideIn: 'slideIn 0.3s ease-in-out'
```

Comment:
these values won't be made available to consumers of the library. They would need to be added to our tailwind plugin for that to happen.

However I think this should really be done with svelte transitions instead - https://svelte.dev/tutorial/adding-parameters-to-transitions